docs/tables/snowflake_resource_monitor.md

```diff
@@ -0,0 +1,45 @@
+# Table: snowflake_warehouse
+
+A warehouse is a cluster of compute resources in Snowflake. Warehouses provide the required resources, such as CPU, memory, and temporary storage, to perform queries.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  name,
+  credit_quota,
+  used_credits,
+  remaining_credits
+from
+  snowflake_resource_monitor;
+```
+
+### List warehouses and % credit left
+
+```sql
+select account as "Account",
+    name as "Warehouse",
+    credit_quota as "Credit Quota",
+    used_credits as "Used Credits",
+    remaining_credits as "Remaining Credits",
+    to_char(used_credits/credit_quota*100,'9990D9') as "% Used",
+    case
+        when used_credits/credit_quota*100 > 90 then 'alert'
+        when used_credits/credit_quota*100 > 75 then 'warning'
+        else 'ok'
+    end as type
+from snowflake_resource_monitor
+where used_credits/credit_quota*100 > 75
+order by used_credits/credit_quota desc
```

Comment:
```suggestion
select
  account,
  name as warehouse,
  credit_quota,
  used_credits,
  remaining_credits,
  to_char(used_credits/credit_quota*100, '9990D9') as percent_used,
  case
    when used_credits/credit_quota*100 > 90 then 'alert'
    when used_credits/credit_quota*100 > 75 then 'warning'
    else 'ok'
  end as type
from
  snowflake_resource_monitor
where
  used_credits/credit_quota*100 > 75
order by
  used_credits/credit_quota desc;
```

My suggestion follows formatting standards as per https://steampipe.io/docs/develop/table-docs-standards#style-conventions-1, if you have any questions on the suggestion please let me know.

Also, for `to_char(used_credits/credit_quota*100, '9990D9')`, will this produce any different results than `round(used_credits/credit_quota*100, 1)`? The `round` function seems a bit simpler to me, but I'm not that familiar with the `9990D9` char format.

Reply:
Done